src/main/java/net/dries007/tfc/common/blocks/wood/ITFCHangingSignBlock.java

```diff
@@ -0,0 +1,7 @@
+package net.dries007.tfc.common.blocks.wood;
+
+import net.minecraft.resources.ResourceLocation;
+
+public interface ITFCHangingSignBlock {
+    ResourceLocation metal();
```

Comment:
Not necessary

Reply:
discussed on discord - using the marker interface is maybe preferable to doing instanceof checks for both TFCCeilingHangingSignBlock and TFCWallHangingSignBlock. but the handling of the resource will be different

Reply:
split up into gui texture and renderer material